app/assets/stylesheets/custom.scss

```diff
@@ -1,45 +1,75 @@
+// fonts
+
+@import url(https://fonts.googleapis.com/css?family=Slabo+27px);
+
+
 .navbar {
   margin-top: 0;
   margin-bottom: 0;
 }
 
 body {
-  background: image-url("raw/goofy-dad-bg.jpg") no-repeat center center fixed;
+  background: image-url("goofy-dad-bg.jpg") no-repeat center center fixed;
   -webkit-background-size: cover;
   -moz-background-size: cover;
   -o-background-size: cover;
   background-size: cover;
 }
 
+// general stuff
+
+.col-centered{
+    float: none;
+    margin: 0 auto;
+}
+
+.item-title {
+  font-size: 1.5em;
+}
 
 // HEADER STUFF
 .dad-header {
-  height: 50vh;
+  height: 40vh;
   text-align: center;
 }
 
 .dad-header h1 {
   color: #ffffff;
   font-size: 5.2em;
+  font-family: 'Slabo 27px', serif;
   font-weight: 1000;
   position: relative;
-  top: 30%;
+  top: 15%;
+  text-shadow: 0px 0px rgba(0,0,0, 0.7);
+}
+.dad-header h3 {
+  border-bottom: 0px;
+  color: #ffffff;
+  font-size: 2.2em;
```

Comment:
Properties should be ordered border-bottom, color, font-family, font-size, font-weight, position, text-shadow, top